bftengine/src/communication/TlsTCPCommunication.cpp

```diff
@@ -971,12 +972,12 @@ class TlsTCPCommunication::TlsTcpImpl {
       tcp::endpoint ep(address::from_string(_listenIp), _listenPort);
       _pAcceptor = boost::make_unique<tcp::acceptor>(_service, ep);
       start_accept();
-    } else // clients donn't listen
+    } else // clients don't listen
       LOG_INFO(_logger, "skipping listen for node: " << _selfId);
 
     // this node should connect only to nodes with lower ID
     // and all nodes with higher ID will connect to this node
-    // we dont want that clients will connect to another clients
+    // we don't want that clients will connect to other clients
```

Comment:
nit: "we don't want that clients will connect to other clients" still sounds a little off to me somehow; I might suggest "we don't want clients to connect to other clients".